### DISCUSSION TITLE
K3S is deployed on K8S, and an error occurs after the restart

### DISCUSSION THREAD

**Discussion by misteruly** [k3s-io/k3s#4028] [Q&A]
## The first initialized master reported an error
k3s version v1.21.2+k3s1 (5a67e8dc)

{"level":"warn","ts":"2021-09-16T06:08:48.023Z","caller":"embed/config_logging.go:279","msg":"rejected connection","remote-addr":"10.233.97.102:32882","server-name":"","error":"remote error: tls: bad certificate"}

**Comment by brandond** [comment]
Yes, that's a normal message to see when a node is starting up, as it makes several tests to verify the cluster CA certificates. Are you seeing any other problems?

**Comment by misteruly** [comment]
After the restart, the certificate content is changed. How can I join the original cluster?






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "k3s-io/k3s"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月16日(星期四) 下午4:22
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [k3s-io/k3s] K3S is deployed on K8S, and an error occurs after the restart (Discussion #4028)





 
Yes, that's a normal message to see when a node is starting up, as it makes several tests to verify the cluster CA certificates. Are you seeing any other problems?
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by brandond** [reply]
What exactly are you restarting? Which certificate is changed?
